pkgs/tools/misc/parallel/default.nix

```
@@ -14,6 +14,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ makeWrapper ];
   buildInputs = [ perl procps ];
 
+  postPatch = lib.optionalString (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    substituteInPlace Makefile.in --replace "$(DESTDIR)$(bindir)/parallel --shell-completion" "${buildPackages.parallel}/bin/parallel --shell-completion"
```

```suggestion
    substituteInPlace Makefile.in \
      --replace '$(DESTDIR)$(bindir)/parallel --shell-completion' '${lib.getExe buildPackages.parallel} --shell-completion'
```

and add a mainProgram to parallel

---

Done